sites/partners/src/lib/applications/formatApplicationData.ts

```diff
@@ -85,42 +85,30 @@ export const mapFormToApi = ({
     ? data.application?.language
     : null
 
-  const submissionDate: Date | null = (() => {
-    // rename default (wrong property names)
-    const {
-      day: submissionDay,
-      month: submissionMonth,
-      year: submissionYear,
-    } = data.dateSubmitted || {}
-    const { hours, minutes = 0, seconds = 0, period } = data?.timeSubmitted || {}
+  const getDateTime = (date: DateFieldValues, time: TimeFieldValues) => {
```

Comment:
**suggestion:** could we pull this out to be a stand alone function so at a later time we can add unit tests